app/views/layouts/_sidebar.html.erb

```diff
@@ -49,6 +49,7 @@
           <%= sidebar_link_to "Linux", 'agent/v3/linux' %>
           <%= sidebar_link_to "Docker", 'agent/v3/docker' %>
           <%= sidebar_link_to "AWS", 'agent/v3/aws' %>
+          <%= sidebar_link_to "Elastic CI Stack", 'agent/v3/elastic_ci' %>
```

Comment:
maybe `Elastic CI Stack for AWS`?

In the future we might have CI stacks for alternative cloud providers, and is so they'll probably be different enough to require their own docs page.

Reply:
Done

Reply:
Shall we rename the file also? Probably best